.github/workflows/deno.yml

```diff
@@ -0,0 +1,42 @@
+name: Deno
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  denofmt-and-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+      - uses: actions/checkout@v3
+
+      - run: deno lint
+      - run: deno fmt --check
```

Comment:
They're _instant_, separate job is unnecessary.

Reply:
but they can fail independently so when the fmt fails the tests can still run through and vice versa. Personally I think its annoying to push the fmt and then see it failing again because of tests directly after.